frontend/src/Components/HomePage/HomePage.css

```diff
@@ -0,0 +1,18 @@
+.header {
+   width: 100%;
+   height: var(--header-height);
+   position: fixed;
+   top: 0;
+   left: 0;
+   display: flex;
+   align-items: center;
+   justify-content: space-between;
+   padding: 0 1rem;
+   background-color: var(--latte);
+   transition: 0.5s;
+ }
+ 
+ .admin {
+  color: var(--gold)!important;
```

Comment:
Try to avoid !important as much as you can, because it is hard to maintain css that relies on important.